infra/terraform/test_fixtures/terraform-google-iam.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+  // Roles required by CI service accounts to run the IAM
+  // integration tests.
+  iam_required_roles = [
+    "roles/resourcemanager.organizationAdmin",
+    "roles/compute.xpnAdmin",
+    "roles/storage.admin",
+    "roles/pubsub.admin",
+    "roles/cloudkms.admin",
+  ]
+}
+
+resource "google_project" "iam" {
+  provider = "google.phoogle"
+  name = "ci-project-factory"
+  project_id = "ci-project-factory"
+  folder_id = "${google_folder.phoogle_cloud_foundation_cicd.name}"
+  billing_account = "${module.variables.phoogle_billing_account}"
+}
+
+resource "google_project_services" "iam" {
+  provider = "google.phoogle"
+
+  project = "${google_project.iam.id}"
+  services = [
```

Comment:
Are we sure that this is the correct set of services to enable?

Reply:
Unfortunately - no, I can't be sure this list is sufficient, I just think it's quicker way to get actual list by running pipeline and satisfy missing requirements, other option - do it on theoretical level by reading documentation and making guesses, as long I don't have enough permissions to run tests from my phoogle account